TeamSnapSDK/SDK/Cache/TSPCache.m

```diff
@@ -22,16 +22,26 @@ +(void)setCacheRootPath:(NSURL *)rootPath {
 
 +(NSURL *)cacheRootPath {
     if (!_rootPath) {
-        NSString *directory = [NSSearchPathForDirectoriesInDomains(NSDocumentDirectory, NSUserDomainMask, YES) lastObject];
+        NSString *directory = [NSSearchPathForDirectoriesInDomains(NSCachesDirectory, NSUserDomainMask, YES) lastObject];
         _rootPath = [[NSURL fileURLWithPath:directory isDirectory:YES] URLByAppendingPathComponent:@"teamsnap"];
     }
     if (![[self fileManager] fileExistsAtPath:[_rootPath path]]) {
         [[self fileManager] createDirectoryAtPath:[_rootPath path] withIntermediateDirectories:YES attributes:nil error:nil];
+        [self markDirectoryForNoBackup:_rootPath];
```

Comment:
@larsacus I read guideline 4 differently as to imply the flag would prevent Caches from being purged in Low-memory situations. Similar to this SO question: https://stackoverflow.com/questions/16601125/does-the-do-not-back-up-attribute-work-on-data-in-library-caches